scripts/performance/tests/__init__.py

```diff
@@ -0,0 +1,24 @@
+class BaseBenchmarkSuite:
```

Comment:
Do we need a base class if there's only one implementation of it? 

If keeping, could you add class-level comments when when someone would use this?